include/swift/Option/FrontendOptions.td

```diff
@@ -456,5 +456,8 @@ def validate_tbd_against_ir_EQ: Joined<["-"], "validate-tbd-against-ir=">,
 // This is used to guard preemptive testing for the fix-it.
 def fix_string_substring_conversion: Flag<["-"], "fix-string-substring-conversion">,
     HelpText<"Emit a fix-it to append '[]' to String expressions when converting to Substring.">;
+    
+def bypass_batch_mode_checks: Flag<["-"], "bypass-batch-mode-checks">,
+    HelpText<"Bypass checks for batch-mode errors.">;
```

Comment:
Is this different from just `-enable-batch-mode` that @graydon added?

Reply:
Yes. Graydon's flags allow or disallow batch mode for the driver. This flag is for experimental testing of batch mode when it would otherwise fail assertions. What do you suggest?

Reply:
I wasn't sure if you were going to have a batch mode flag for the frontend or not. If you're going to have one anyway, you might as well use the same one that the driver uses. Is there going to be any difference between batch mode with exactly one primary file and classic single-primary-file mode?

Reply:
In the end, there won't be a batch mode flag for the frontend. This one is just for testing until it's done. No, there won't be any difference--batch mode with one primary should be exactly what happens now. The purpose of this flag is to see what breaks when passing in two primaries in a context where that has not been implemented yet. I see there is a flag "enable_experimental_property_behaviors." Would it serve to add the word "experimental" to this flag?

Reply:
Either way is okay with me. But it sounds like @graydon should un-mark his options from being FrontendFlags.